owncloudTestUtil/build.gradle

```diff
@@ -0,0 +1,25 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+android {
+    compileSdkVersion 28
```

Comment:
why don't include compileSdkVersion and minSdkVersion in the root build.gradle file and use them in every child like this??

Reply:
Done